src/main/java/de/doubleslash/keeptime/view/ReportController.java

```diff
@@ -176,7 +164,37 @@ protected void updateItem(final TableRow item, final boolean empty) {
       this.workTableTreeView.getColumns().add(timeRangeColumn);
 
       final TreeTableColumn<TableRow, String> timeSumColumn = new TreeTableColumn<>("Duration");
-      timeSumColumn.setCellValueFactory(new TreeItemPropertyValueFactory<TableRow, String>("timeSum"));
+      timeSumColumn.setCellFactory(new Callback<>() {
+         @Override
+         public TreeTableCell<TableRow, String> call(TreeTableColumn<TableRow, String> tableRowStringTreeTableColumn) {
+
+            return new TreeTableCell<>() {
+
+               @Override
+               protected void updateItem(String timeString, boolean empty) {
+                  super.updateItem(timeString, empty);
+
+                  if (timeString == null || empty) {
+                     this.setGraphic(null);
+                     this.setText(null);
+                  } else {
+                     this.setGraphic(new Label(timeString));
+
+                     for (String workTime : workTime) {
```

Comment:
why do we need a for-loop here? not sure what this logic does actually. It should set the "timeString" as underline I guess? We should not reference external variables. Only the ones from updateItem method.

Reply:
I need the external variable to check if a time is a work time or a regular. Its the only way I found to find out if a time belongs to a work item.

Reply:
The forloop is needed to underline all working times